code/scripting/api/libs/mission.cpp

```diff
@@ -1513,6 +1513,21 @@ ADE_FUNC(hasCommandBriefing,
 	return ade_set_args(L, "b", mission_has_cmd_brief() != 0);
 }
 
+ADE_FUNC(hasDebriefing,
+	l_Mission,
+	nullptr,
+	"Determines if the current mission has a command briefing",
+	"boolean",
+	"true if command briefing, false otherwise.")
+{
+	bool check = false;
+	if (The_mission.flags[Mission::Mission_Flags::Toggle_debriefing]) {
+		check = true;
```

Comment:
Shouldn't this check default to `true`?  Most missions have debriefings.

Reply:
Yes. Good catch